multi-agent/adapters/inbound/flask/endpoints/sessions.py

```
@@ -305,6 +317,195 @@ def list_active_streams():
         return jsonify({"error": str(e)}), 500
 
 
+@sessions_bp.route("/session.approval", methods=["POST"])
+@require_authentication
+@from_body({
+    "session_id": fields.Str(data_key="sessionId", required=True),
+    "request_id": fields.Str(data_key="requestId", required=True),
+    "decision": fields.Str(data_key="decision", required=True),
+    "feedback": fields.Str(data_key="feedback", load_default=""),
+    "modified_args": fields.Dict(data_key="modifiedArgs", load_default=lambda: {}),
```

Ruff (0.15.20)

[warning] Prefer `dict` over useless lambda

Replace with `lambda` with `dict`

(PIE807)